src/freenas/usr/local/libexec/collectd_alert.py

```diff
@@ -46,9 +46,9 @@ def main():
 
     data = {}
     if os.path.exists(COLLECTD_FILE):
```

Comment:
Let's go ahead and fix the 2 issues here.

1. this is a `flake8` error because it's an empty `except:` statement
2. let's remove the check for `os.path.exists` before opening this file for reading since it's prone to a race condition. Instead we should use `contextlib.suppress(FileNotFoundError)` context manager and then try to open for reading.